### ISSUE TITLE
Roku Music Not Displaying Genre

### ISSUE THREAD

**Issue by jwor80** [jellyfin-archive/jellyfin-roku-legacy#807]
### Please describe your bug

Running JellyFin 1.5 on a Roku Express.

Steps to Reproduce:
Browse to my Music Library
Click the Options
Select View
Under View I have an option of Default, Artists, or Albums.

If I use these same steps for Movies or TV Shows I have a Genre option listed under the View Menu.
If I use a web browser to view my music I'm able to sort by Genre

### Jellyfin Version

10.8.0

### if other:

10.8.4

### Environment

```markdown
- OS: Windows
- Virtualization: None
- Clients: Roku
- Browser: Chrome
- FFmpeg Version: 10.8.4-Jellyfin
- Playback Method: N/A
- Hardware Acceleration: N/A
- Plugins: N/A
- Reverse Proxy: N/A
- Base URL: N/A
- Networking: N/A
- Storage: N/A
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by 1hitsong** [comment]
Adding all the views and filters from the web app into Roku is on our roadmap: https://github.com/jellyfin/jellyfin-roku/discussions/662

This was only the first release with music support, so we're still working on what we call the foundational features before we start moving into more of the "nice to haves."
